README.md

```diff
@@ -1,136 +1,138 @@
 ![Banner](./GitHub_Banner.png)
 
-## Finamplify - Finamp's First Hackathon!
 
-From Saturday, March 29th (2025-03-29) until the end of Sunday, April 6th (2025-04-06), we are holding Finamp's first hackathon - called *Finamplify*.  
-This is 9 days of intensive work on Finamp, by new and existing contributors!
+> [!NOTE]
+> Finamp is currently being reworked and redesigned! Instructions on how to join the beta are included in every [release](https://github.com/jmshrv/finamp/releases)!
 
-Here are the most important infos:
+## TL;DR
+Finamp is a free Jellyfin music player for Android, iOS and Desktop. It allows you to Stream and Download the music you own. The design is mimicking popular music streaming services but still with a unique twist and a few customizations and of course privacy!
+> [!WARNING]
+> You do need access to an [jellyfin](https://jellyfin.org) server else you wont be able to use Finamp at all.
```

Comment:
I'd like to have this as a note, not a warning. Maybe use `[!IMPORTANT]` instead?